include/algorithms/logistic_regression/logistic_regression_predict_types.h

```diff
@@ -226,6 +226,9 @@ typedef services::SharedPtr<const Result> ResultConstPtr;
 
 } // namespace interface1
 using interface1::Input;
+using classifier::Parameter;             /* Support of static backward compatibility */
```

Comment:
don`t forget to make related changes in the testing branch and static BC test branches.